### PR TITLE
print used attributes

### LINKED ISSUE #16: Feature to extract used win32more module names
Hugely depends on #15 (it would be easy to implement using metaclass)

In my test project, with pyc files gathered by pyinstaller: 9.45 Megabytes
Then collect all used functions in one place and drop `win32more.Windows.*`: 7.17 KiloBytes

### PR DESCRIPTION
**Pull request by T-256**
closes #16

### CODE REVIEW

File: win32more/__init__.py [hunk 1]
```diff
@@ -441,18 +489,49 @@ def __commit__(self):
         return self._prototype()
 
 
+class ConstantWithAnnotations:
+    def __init__(self, prototype, annotation, module):
+        self._prototype = prototype
+        # Maybe generator could do this for us, like func/struct parameters hints.
```
Comment (T-256) [inline win32more/__init__.py:495]: is it possible `generate.py` write full path annotations for constants?

File: win32more/__init__.py [hunk 2]
```diff
@@ -441,18 +489,49 @@ def __commit__(self):
         return self._prototype()
 
 
+class ConstantWithAnnotations:
+    def __init__(self, prototype, annotation, module):
+        self._prototype = prototype
+        # Maybe generator could do this for us, like func/struct parameters hints.
+        if annotation not in EXCLUDED_ATTRS and not annotation.startswith('win32more.'):
+            annotation = f"{module.__name__}.{annotation}"
```
Comment (T-256) [inline win32more/__init__.py:497]: is it possible `generate.py` write full path annotations for constants?

Comment (ynkdir) [reply win32more/__init__.py:497]: no problem.

**Comment by T-256** [comment]
External solution: https://github.com/ynkdir/py-win32more/issues/16#issuecomment-1763257872

**Pull request closed without merging**